### PR TITLE
[WIP]Feat/ada3 feedback

### PR DESCRIPTION
**Pull request by 8ball030**
## This removes all files with docs.fetch.ai from our docs.

image

I would think we can do better and I have removed the SOEF information.

There is a file modified, ```protocol``` which i think we could do with re-implementing using a different protocol, perhaps the ABCI, or potentially contract_api dialogues

Note this removes the entire protocol walkthrough as it is based on the fetchai repo.

Here is an issue to potentially replace it;

https://github.com/valory-xyz/open-aea/issues/90

### CODE REVIEW

File: docs/interaction-protocol.md [hunk 1]
```diff
@@ -30,15 +26,6 @@ The `fetchai/http_client` connection can be used to make requests to third party
 
 Note that in general, third party SDKs can be wrapped in a connection and shared with other developers as a package. Often this also involves creating a custom protocol to enforce the type of interactions permitted between skills and the connection wrapping the SDK.
 
-
-## Next steps
-
-### Recommended
```
Comment (DavidMinarsch) [inline docs/interaction-protocol.md:36]: Recommend something else then to not break the structure?

File: docs/protocol.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Protocols in the AEA world provide definitions for:
...
 
-The framework provides a `default` protocol. This protocol provides a bare-bones implementation for an AEA protocol which includes a <a href="https://docs.fetch.ai/aea/api/protocols/default/message/#packages.fetchai.protocols.default.message" target="_blank">`DefaultMessage`</a>  class and associated <a href="https://docs.fetch.ai/aea/api/protocols/default/serialization/#packages.fetchai.protocols.default.serialization" target="_blank">`DefaultSerializer`</a> and <a href="https://docs.fetch.ai/aea/api/protocols/default/dialogues/#packages.fetchai.protocols.default.dialogues" target="_blank">`DefaultDialogue`</a> classes.
+The framework provides a `base` protocol. This protocol provides a bare-bones implementation for an AEA protocol which includes a <a href="https://valory-xyz.github.io/open-aea/api/protocols/base/#message-objects" target="_blank">`DefaultMessage`</a>  class and associated <a href="https://valory-xyz.github.io/open-aea/api/protocols/base/#serializer-objects" target="_blank">`DefaultSerializer`</a> and <a href="https://valory-xyz.github.io/open-aea/api/protocols/dialogue/base/" target="_blank">`DefaultDialogue`</a> classes.
```
Comment (DavidMinarsch) [inline docs/protocol.md:13]: please do relative paths

File: docs/decision-maker.md [hunk 3]
```diff
@@ -6,8 +6,6 @@ Skills communicate with the decision maker via <a href="../api/protocols/base#me
 
 - <a href="../api/protocols/signing/message#signingmessage-objects">`SigningMessage`</a>: it is used by skills to propose a transaction to the decision-maker for signing.
 
-- <a href="https://docs.fetch.ai/aea/api/protocols/state_update/message/#stateupdatemessage-objects" target="_blank">`StateUpdateMessage`</a>: it is used to initialize the decision maker with preferences and ownership states. It can also be used to update the ownership states in the decision maker if the settlement of transaction takes place.
```
[the remark this answers is not in the document]
Comment (8ball030) [reply docs/decision-maker.md:9]: No we dont have docs for this however we do have the code for the protocol. 
I can look into getting those docs generated as well.

[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply docs/decision-maker.md:9]: ok nm

File: docs/build-aea-step-by-step.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ Building an AEA step by step (ensure you have followed the <a href="../quickstar
 	`aea search connections`, then `aea add connection [public_id]`
 </li>
 <li>Look for, then add or generate the <a href="../protocol/">protocols</a> you require: `aea search protocols`, then `aea add protocol [public_id]` or `aea generate protocol [path_to_specification]`</li>
-<li>Look for, then add or code the <a href="../skill/">skills</a> you need: `aea search skills`, then `aea add skill [public_id]`. <a href="https://docs.fetch.ai/aea/skill-guide/" target="_blank">This guide</a> shows you step by step how to develop a skill.</li>
+<li>Look for, then add or code the <a href="../skill/">skills</a> you need: `aea search skills`, then `aea add skill [public_id]`. <a href="https://valory-xyz.github.io/open-aea/skill/" target="_blank">This guide</a> shows you step by step how to develop a skill.</li>
```
Comment (DavidMinarsch) [inline docs/build-aea-step-by-step.md:9]: All our docs paths should be relative paths

File: mkdocs.yml [hunk 5]
```diff
@@ -82,11 +82,6 @@ nav:
     - Agent Communication:
       - Agent Communication Network: 'acn.md'
       - ACN Internals: 'acn-internals.md'
-    - Search & Discovery:
-      - Simple OEF: 'simple-oef.md'
-      - Defining Data Models: 'defining-data-models.md'
-      - The Query Language: 'query-language.md'
```
Comment (DavidMinarsch) [inline mkdocs.yml:88]: These two files should also be deleted then

File: tests/test_docs/test_docs_protocol.py [hunk 6]
```diff
@@ -1,115 +0,0 @@
...
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-#
-# ------------------------------------------------------------------------------
-
-"""This module contains the tests for the code-blocks in the protocol.md file."""
```
Comment (angrybayblade) [inline tests/test_docs/test_docs_protocol.py:21]: Removed, since there are no code blocks in the protocol.md file

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**